packages/ember-runtime/lib/computed/reduce_computed_macros.js

```diff
@@ -278,6 +278,28 @@ export function mapBy(dependentKey, propertyKey) {
   hamster.get('remainingChores'); // [{name: 'write more unit tests', done: false}]
   ```
 
+  You can also use `@each.property` in your dependent key, the callback will still use the underlying array:
+
+  ```javascript
+  let Hamster = Ember.Object.extend({
+    remainingChores: Ember.computed.filter('chores.@each.done', function(chore, index, array) {
+      return !chore.get('done');
+    })
+  });
+
+  let hamster = Hamster.create({
+    chores: [
+      { name: 'cook', done: true },
+      { name: 'clean', done: true },
+      { name: 'write more unit tests', done: true }
+    ]
+  });
+  hamster.set('done', false);
```

Comment:
What is this setting? should the example be to set the last item of `chores` as `done: false`? or perhaps just remove the `set` example and define one of the chores to have `done: false` in the call to `Hamster.create`